core/protobuf/src/proto/dataserver/protocol.proto

```diff
@@ -88,9 +90,10 @@ message Heartbeat {
 
 // Netty RPCs. Every RPC needs to define a request type and optionally a response type (default to Response).
 
-// next available id: 3
+// next available id: 4
```

Comment:
Yeah it was strange that it skipped one, but we should be okay to reclaim the space.